### ISSUE TITLE
NVDA add-on: unable to navigate among the categories of MathCAT settings

### ISSUE THREAD

**Issue by jasonjgw** [NSoiffer/MathCATForPython#26]
# Steps to Reproduce
1. Navigate to Tools -> MathCAT Settings Focus should be on the list of setting categories, with "braille" current.

2. Use the Down-Arrow key in an attempt to move to the next item in the list of categories.

# Expected Result
The next category should become current, and focus should remain on the category list.

# Actual Result
Focus moves to the combo box to change the braille translation option (i.e., the braille code), and thus moves away from the category list altogether.

This is a regression.

Tested with NVDA 2022.1 beta 2.

**Comment by jasonjgw** [comment]
I can also reproduce the issue with NVDA 2022.1 beta 3.
